scoped-transitions.md

```diff
@@ -0,0 +1,110 @@
+# Scoped Transitions
+
+This doc is a summary of early design explorations to scope transitions to a DOM sub-tree. The following is a rough API sketch for how a developer would trigger the transition rooted on an element (other than the document's root element).
+
+```js
+element.createTransition({
+  updateDOM() {
+    /* … */
+  },
+});
+```
+
+The element becomes the **scoped-transition-root** for the transition. It's the element that will host the pseudo-element tree and page-transition-container sub-trees will be created for descendants that have a `page-transition-tag`. For example, the following page:
```

Comment:
I'd add a disclaimer right at the top saying that this document talks about the pseudo-element tree as the implementation of the scoped transitions. Some considerations for a different implementation, Shadow DOM, are considered at the end

Reply:
Done.